config/data/holes.toml

```diff
@@ -1128,6 +1128,425 @@ preamble = '''
+    "Serbia":                           ["RS", "🇷🇸"], "Seychelles":                       ["SC", "🇸🇨"],
+    "Sierra Leone":                     ["SL", "🇸🇱"], "Singapore":                        ["SG", "🇸🇬"],
+    "Sint Maarten":                     ["SX", "🇸🇽"], "Slovakia":                         ["SK", "🇸🇰"],
+    "Slovenia":                         ["SI", "🇸🇮"], "Solomon Islands":                  ["SB", "🇸🇧"],
+    "Somalia":                          ["SO", "🇸🇴"], "South Africa":                     ["ZA", "🇿🇦"],
+    "South Georgia and the South Sandwich Islands":                                       ["GS", "🇬🇸"],
+    "South Korea":                      ["KR", "🇰🇷"], "South Sudan":                      ["SS", "🇸🇸"],
+    "Spain":                            ["ES", "🇪🇸"], "Sri Lanka":                        ["LK", "🇱🇰"],
+    "Sudan":                            ["SD", "🇸🇩"], "Suriname":                         ["SR", "🇸🇷"],
+    "Sweden":                           ["SE", "🇸🇪"], "Switzerland":                      ["CH", "🇨🇭"],
+    "Syria":                            ["SY", "🇸🇾"], "Taiwan":                           ["TW", "🇹🇼"],
+    "Tajikistan":                       ["TJ", "🇹🇯"], "Tanzania":                         ["TZ", "🇹🇿"],
+    "Thailand":                         ["TH", "🇹🇭"], "Timor-Leste":                      ["TL", "🇹🇱"],
+    "Togo":                             ["TG", "🇹🇬"], "Tokelau":                          ["TK", "🇹🇰"],
+    "Tonga":                            ["TO", "🇹🇴"], "Trinidad and Tobago":              ["TT", "🇹🇹"],
+    "Tristan da Cunha":                 ["TA", "🇹🇦"], "Tunisia":                          ["TN", "🇹🇳"],
+    "Türkiye":                          ["TR", "🇹🇷"], "Turkmenistan":                     ["TM", "🇹🇲"],
+    "Turks and Caicos Islands":         ["TC", "🇹🇨"], "Tuvalu":                           ["TV", "🇹🇻"],
+    "Uganda":                           ["UG", "🇺🇬"], "Ukraine":                          ["UA", "🇺🇦"],
+    "United Arab Emirates":             ["AE", "🇦🇪"], "United Kingdom":                   ["GB", "🇬🇧"],
+    "United Nations":                   ["UN", "🇺🇳"], "United States":                    ["US", "🇺🇸"],
+    "United States Virgin Islands":     ["VI", "🇻🇮"], "Uruguay":                          ["UY", "🇺🇾"],
+    "Uzbekistan":                       ["UZ", "🇺🇿"], "Vanuatu":                          ["VU", "🇻🇺"],
+    "Vatican City":                     ["VA", "🇻🇦"], "Venezuela":                        ["VE", "🇻🇪"],
+    "Vietnam":                          ["VN", "🇻🇳"], "Wallis and Futuna":                ["WF", "🇼🇫"],
+    "Western Sahara":                   ["EH", "🇪🇭"], "Yemen":                            ["YE", "🇾🇪"],
+    "Zambia":                           ["ZM", "🇿🇲"], "Zimbabwe":                         ["ZW", "🇿🇼"]
+}'''
+preamble = '''<div id="flags" class=tables-4col>
+    <table>
```

Comment:
You can loop over `.DataMap` to avoid repeating the data here, just look at how CSS Colors does it, you basically need to pick the names of when you want to break into the next table.